healthtools/scrapers/health_facilities.py

```diff
@@ -1,23 +1,13 @@
 import json
+import math
 from healthtools.scrapers.base_scraper import Scraper
 from healthtools.config import SMALL_BATCH_HF
 import requests
 from datetime import datetime
 
 TOKEN_URL = "http://api.kmhfl.health.go.ke/o/token/"
-SEARCH_URL = "http://api.kmhfl.health.go.ke/api/facilities/material/?page_size={}&" \
-             "fields=id,regulatory_status_name,facility_type_name,facility_type_parent,owner_name,owner_type_name," \
-             "owner_type,operation_status_name,county,constituency,constituency_name,ward_name,average_rating," \
-             "facility_services,is_approved,has_edits,latest_update,regulatory_body_name,owner,date_requested," \
-             "date_approved,latest_approval_or_rejection,sub_county_name,sub_county_id,county_name,constituency_id," \
-             "county_id,keph_level_name,facility_contacts,coordinates,lat_long,latest_approval,county_code,constituency_code" \
-             ",ward_code,service_catalogue_active,facility_units,officer_in_charge,created,updated,deleted,active,search," \
-             "name,official_name,code,registration_number,abbreviation,description,number_of_beds,number_of_cots," \
-             "open_whole_day,open_public_holidays,open_normal_day,open_weekends,open_late_night,is_classified," \
-             "is_published,regulated,approved,rejected,bank_name,branch_name,bank_account,facility_catchment_population," \
-             "town_name,nearest_landmark,plot_number,location_desc,closed,closed_date,closing_reason,date_established," \
-             "license_number,created_by,updated_by,facility_type,operation_status,ward,parent,regulatory_body," \
-             "keph_level,sub_county,town,regulation_status,contacts&format=json"
+SEARCH_URL = "http://api.kmhfl.health.go.ke/api/facilities/material/\
+?format=json&page={}&page_size=5000"
```

Comment:
Does this still return all the fields?

Reply:
Yes it does. And it sets the size of data each page returns to 5000